alchemist/alchemist-cognitive-agents/src/main/kotlin/it/unibo/alchemist/model/implementations/actions/FollowFlowField.kt

```diff
@@ -15,21 +15,21 @@ import it.unibo.alchemist.model.interfaces.environments.EuclideanPhysics2DEnviro
  * @param targetMolecule
  *          the {@link Molecule} you want to know the concentration in the different positions of the environment.
  */
-open class FollowGradient<T>(
-    env: EuclideanPhysics2DEnvironment<T>,
-    reaction: Reaction<T>,
-    pedestrian: Pedestrian2D<T>,
+open class FollowFlowField(
+    env: EuclideanPhysics2DEnvironment<Double>,
+    reaction: Reaction<Double>,
+    pedestrian: Pedestrian2D<Double>,
     targetMolecule: Molecule
-) : GradientSteeringAction<T>(
+) : FlowFieldSteeringAction<Double>(
```

Comment:
Can we switch to "Number"?

Reply:
The only problem is that numbers are not comparable by default and I need to compare the different concetrations to determine the next position.

Reply:
You can still use number, then call `toDouble()` and compare doubles. This way, works for a larger set of types.

Reply:
fixed